hikari/guilds.py

```diff
@@ -246,6 +247,23 @@ class GuildVerificationLevel(int, enums.Enum):
     """Must have a verified phone number."""
 
 
+@typing.final
+class GuildNSFWLevel(int, enums.Enum):
+    """Represents the NSFW level of a guild."""
+
+    DEFAULT = 0
+    """Default level of NSFW (per channel)."""
+
+    EXPLICIT = 1
+    """Guild contains explicit NSFW content."""
+
+    SAFE = 2
+    """Guild is safe of NSFW content."""
+
+    AGE_RESTRICTED = 2
+    """Guild is age restricted due to NSFW content."""
```

Comment:
Could this go into more detail about this actually means and maybe link to a support article about it?